testing/bubbleSort.js

```diff
@@ -0,0 +1,10 @@
+import { expect } from 'chai'
+import bubbleSort from '../functions/bubbleSort.js'
+
+describe('#bubbleSort', () => {
+  context('takes an array of unsorted integers', () => {
+    it('should return an array of sorted integers using bubbleSort', ()=>{
+      expect(bubbleSort([5,1,4,2,8])).to.eql([1,2,4,5,8])
```

Comment:
It's up to you, but I prefer using `to.deep.equal` instead of `to.eql` because it actually tells you that it's doing a deep equality check instead of just removing vowels from the shallow equality check function.